project/plugins.sbt

```diff
@@ -6,7 +6,7 @@ addSbtPlugin("uk.gov.hmrc" % "sbt-auto-build" % "3.4.0")
 
 addSbtPlugin("uk.gov.hmrc" % "sbt-distributables" % "2.1.0")
 
-addSbtPlugin("com.typesafe.play" % "sbt-plugin" % "2.7.9")
+addSbtPlugin("com.typesafe.play" % "sbt-plugin" % "2.8.7")
```

Comment:
Note: the latest version is `2.8.8` at the moment.

Reply:
ok - want me to update it?

Reply:
@joesheridan Up to your discretion. Probably better to update.

Reply:
ok done